src/mrack/providers/static.py

```diff
@@ -82,4 +87,5 @@ def prov_result_to_host_data(self, prov_result):
 
     async def delete_host(self, host_id):
         """Fake delete - pass but don't do anything."""
+        logger.info(f"{self.dsp_name}: Removing static host {host_id}")
```

Comment:
Static host cannot be removed by mrack. Thus the log information needs to say something like: "Host {host_id} is static - not managed by Mrack, ignoring removal."

Reply:
Will edit